testlib/helper.go

```diff
@@ -122,6 +123,10 @@ func (h *MainHelper) Close() error {
 		os.RemoveAll(h.testResourcePath)
 	}
 
+	if r := recover(); r != nil {
```

Comment:
Worth pulling this out as a separate change so we can track its impact discretely?

Reply:
Created a PR: https://github.com/mattermost/mattermost-server/pull/11726